propka/atom.py

```diff
@@ -92,104 +101,148 @@ def set_properties(self, line):
             self.element = line[12:14].strip().strip(string.digits)
             if len(self.name) == 4:
                 self.element = self.element[0]
-            if len(self.element)==2:
-                self.element = '%1s%1s'%(self.element[0], self.element[1].lower())
+            if len(self.element) == 2:
+                self.element = '%1s%1s' % (self.element[0], self.element[1].lower())
 
-        return
+    def set_group_type(self, type_):
+        """Set group type of atom.
 
-    def set_group_type(self, type):
-        self.group_type = type
-        return
+        Args:
+            type_:  group type of atom
+        """
+        self.group_type = type_
 
     def count_bonded_elements(self, element):
+        """Count number of bonded atoms with same element.
+
+        TODO - this function is silly.  It should just be the len() of the
```

Comment:
true, unless these lists get really big (I doubt it) and you don't want the temporary memory consumption

Reply:
Addressed in https://github.com/Electrostatics/propka-3.1/commit/4b3dee54f74f74e3662069c69248793998953929